in-cluster/default.yaml

```diff
@@ -168,6 +168,23 @@ spec:
         name: kots/rqlite/logs
         selector:
           - app=kotsadm-rqlite
+    - logs:
+        collectorName: minio-logs
+        name: kots/minio/logs
+        selector:
+          - app=kotsadm-minio
+    - exec:
```

Comment:
this seems useful, could we get this added for the non kotsadm-minio as well?

Reply:
thanks @hedge-sparrow, do you mean the `minio-fs` for hostpath snapshot? I've updated the spec accordingly.